test/test_maxwellDriver2d.py

```diff
@@ -64,4 +64,29 @@ def test_fdtd2d_te_pec():
     #     # plt.grid(which='both')
     #     # plt.pause(0.01)
     #     # plt.cla()
-    #     driver.step()
\ No newline at end of file
+    #     driver.step()
+
+# def test_fdtd2d_te_pmc():
+#     sp = FDTD2D(x_min=-1.0, x_max=1.0, kx_elem=100, boundary_label="PMC")
+#     driver = MaxwellDriver(sp, timeIntegratorType='LF2', CFL=1.0)
+#     s0 = 0.25
+#     final_time = 2.0
+
+#     xH, yH = np.meshgrid(sp.xH, sp.yH)
+#     initialFieldH = np.exp(-xH**2/(2*s0**2))
+#     driver['H'][:,:] = initialFieldH[:,:]
+
+#     driver.run_until(final_time)
+
+#     finalFieldH = driver['H']
+#     R = np.corrcoef(initialFieldH.ravel(), finalFieldH.ravel())
+#     assert R[0, 1] > 0.9999
```

Comment:
This test is okay, does it pass? I think it shouldn't with your current implementation of PMC you are making the box half step smaller in each direction.